server/re/src/keyValPairMS.cpp

```diff
@@ -1,35 +1,24 @@
-/**
- * @file	keyValPairMS.cpp
- *
- */
-
-/*** Copyright (c), The Regents of the University of California            ***
- *** For more information please refer to files in the COPYRIGHT directory ***/
-//#include "reGlobalsExtern.hpp"
-//#include "reFuncDefs.hpp"
 #include "icatHighLevelRoutines.hpp"
+#include "msParam.h"
 #include "rcMisc.h"
-
 #include "irods_re_structs.hpp"
+#include "rodsErrorTable.h"
 
 /**
- * \fn msiGetValByKey(msParam_t* inKVPair, msParam_t* inKey, msParam_t* outVal, ruleExecInfo_t *rei)
```

Comment:
i think doxygen still wants this line?

Reply:
it doesn't need those lines when the documentation is next to the function. assuming the function name and parameters match.

Reply:
oh?   fine if true.

Reply:
i'll verify if this is needed before the end of this weekend.